docs/proto/proto-docs.md

```diff
@@ -329,6 +331,39 @@ ImportDump has all module data for non seed mode.
 
 
 
+<a name="confio.poe.v1beta1.MixerContractConfig"></a>
+
+### MixerContractConfig
+Mixer contract setup. See
+https://github.com/confio/poe-contracts/tree/main/contracts/tg4-mixer
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `sigmoid` | [MixerContractConfig.Sigmoid](#confio.poe.v1beta1.MixerContractConfig.Sigmoid) |  | Sigmoid returns a sigmoid-like value of staked amount times engagement points. See the Proof-of-Engagement white-paper for details. |
```

Comment:
I would like to support the other variants as well. I can define these on a branch off of this.

Reply:
I will do that later, please merge without that. It is not required yet